src/main/kotlin/no/nav/syfo/varsel/AktivitetskravVarselPlanner.kt

```diff
@@ -119,8 +119,7 @@ class AktivitetskravVarselPlanner(
     }
 
     private fun isGyldigSyketilfelledag(syketilfelledag: Syketilfelledag): Boolean {
-        return (syketilfelledag.prioritertSyketilfellebit?.tags?.contains(SyketilfellebitTag.SYKMELDING.name) == true
-                || syketilfelledag.prioritertSyketilfellebit?.tags?.contains(SyketilfellebitTag.PAPIRSYKMELDING.name) == true)
-                && syketilfelledag.prioritertSyketilfellebit.tags.contains(SyketilfellebitTag.SENDT.name)
+        return (syketilfelledag.prioritertSyketilfellebit?.tags?.contains(SyketilfellebitTag.SYKMELDING.name) == true || syketilfelledag.prioritertSyketilfellebit?.tags?.contains(SyketilfellebitTag.PAPIRSYKMELDING.name) == true)
```

Comment:
Kan gjerne gjøre den litt enklere å lese, f.eks assigne syketilfelledag.prioritertSyketilfellebit?.tags til variabel, og splitte || og && i to ulike variabler.